samples: bluetooth: peripheral_past: Fix uninitialized parameter

aaronyegx opened the pull request:
The uninitialized past_param.options is uncertain and causes bt_le_per_adv_sync_transfer_subscribe() returning error.

> PAST subscribe failed (err -22)

jhedberg reviewed (changes requested):
@aaronyegx the change itself is fine, but could you please make the commit message a bit cleaerer. You can use the same subject as I did for this PR, and don't call it "uncertain", rather "uninitialized" is more accurate.

aaronyegx committed:
samples: Bluetooth: peripheral_past: fix the uninitialized parameter

The past_param.options is uninitialized and causes
bt_le_per_adv_sync_transfer_subscribe() returning error.

Signed-off-by: Aaron Ye <aye@ambiq.com>
